devops/iam-did-auth-proxy-dev/values.yaml

```diff
@@ -74,14 +74,14 @@ did-auth-proxy-helm:
 
   opsValues:
     RPC_URL: https://volta-rpc.energyweb.org/
```

Comment:
In gp4btc-stg rpc url is `https://rpc.energyweb.org` which is probably incorrect https://github.com/energywebfoundation/did-auth-proxy/blob/e382f5bd5e46fb657935d4553b29c0119ebd4577/devops/iam-did-auth-proxy-dev/values.yaml#L72. I chanched this during merging